metrics-facade-dropwizard/src/main/java/com/ringcentral/platform/metrics/dropwizard/samples/DropwizardMetricRegistryDefaultInstanceSamplesProvider.java

```diff
@@ -0,0 +1,213 @@
+
+                    final var defaultInstanceSample = new DefaultInstanceSample();
+                    final var sample = new DefaultSample(name, value, DELTA);
+
+                    defaultInstanceSample.add(sample);
+
+                    if (!defaultInstanceSample.isEmpty()) {
+                        result.add(defaultInstanceSample);
+                    }
+                });
+    }
+
+    private void exportHistograms(final Set<DefaultInstanceSample> result) {
+        metricRegistry.getHistograms()
+                .forEach((name, histogram) -> {
+                    final var defaultInstanceSample = new DefaultInstanceSample();
+
+                    final var countSample = createSample(name, "count", histogram::getCount, DELTA);
+                    defaultInstanceSample.add(countSample);
+
+                    final var snapshot = histogram.getSnapshot();
+
+                    final var meanSample = createSample(name, "mean", snapshot::getMean, INSTANT);
+                    defaultInstanceSample.add(meanSample);
+
+                    final var maxSample = createSample(name, "max", snapshot::getMax, INSTANT);
+                    defaultInstanceSample.add(maxSample);
+
+                    final var minSample = createSample(name, "min", snapshot::getMin, INSTANT);
+                    defaultInstanceSample.add(minSample);
+
+                    final var medianSample = createSample(name, "median", snapshot::getMedian, INSTANT);
+                    defaultInstanceSample.add(medianSample);
+
+                    final var stdDevSample = createSample(name, "std_dev", snapshot::getStdDev, INSTANT);
+                    defaultInstanceSample.add(stdDevSample);
+
+                    final var percentile75Sample = createSample(name, "75_percentile", snapshot::get75thPercentile, INSTANT);
+                    defaultInstanceSample.add(percentile75Sample);
+
+                    final var percentile95Sample = createSample(name, "95_percentile", snapshot::get95thPercentile, INSTANT);
+                    defaultInstanceSample.add(percentile95Sample);
+
+                    final var percentile98Sample = createSample(name, "98_percentile", snapshot::get98thPercentile, INSTANT);
+                    defaultInstanceSample.add(percentile98Sample);
+
+                    final var percentile99Sample = createSample(name, "99_percentile", snapshot::get99thPercentile, INSTANT);
+                    defaultInstanceSample.add(percentile99Sample);
+
+                    final var percentile999Sample = createSample(name, "999_percentile", snapshot::get999thPercentile, INSTANT);
+                    defaultInstanceSample.add(percentile999Sample);
+
+                    if (!defaultInstanceSample.isEmpty()) {
+                        result.add(defaultInstanceSample);
+                    }
+                });
+    }
+
+    private void exportTimers(final Set<DefaultInstanceSample> result) {
+        metricRegistry.getTimers()
```

Comment:
What about Rate.* measurables that are TimerMeasurable?

Reply:
fixed, thank you.